File: src/Dato/Usuario.go
package Dato

type JsonUsuarios struct {
	Usuarios []Usuario `json:"Usuarios"`
}

type Usuario struct {
	Dpi      int64  `json:"Dpi"`
	Nombre   string `json:"Nombre"`
	Correo   string `json:"Correo"`
	Password string `json:"Password"`
	Cuenta   string `json:"Cuenta"`
}

type BusquedaUsuario struct {
	Dpi int64 `json:"Dpi"`
}

File: virtualMallFront/src/app/models/usuario/usuario.ts
export class Usuario {
    Dpi: number 
    Nombre   :string 
	Correo   :string 
	Password :string 
	Cuenta   :string 

    constructor(_dpi: number, nombre :string, correo :string, password: string, cuenta: string){
        this.Dpi=_dpi
        this.Nombre=nombre
        this.Correo=correo
        this.Password = password
        this.Cuenta = cuenta
    }
}

export class BusquedaU {
    Dpi: number 
    constructor(_dpi: number){
        this.Dpi=_dpi

    }
}